### PR TITLE
Added a language layout switching button

### PR DESCRIPTION
**Pull request by Johannes-bichler**
Hello,
This keyboard does support multiple layouts, but does not support switching between them at runtime.
For this purpose, I've implemented a button (LanguageKey), that switches between a List of Layouts that can be set via a property (availableLanguageLayouts) in InputPanel.
As this layout switching was added, the Greek Layout was changed to be similar to the others, as the internal layout switcher
(greek-latin) is no longer needed.
If this property is kept empty, it will be automatically set to En, but the property languageLayout overrides this.
If only one language is selected, the layout changer will not be visible.
If an invalid layout is added to the list property, it will fallback to "En".

### CODE REVIEW

File: src/qml/CsLayout.qml [hunk 1]
```diff
@@ -227,7 +227,12 @@ ColumnLayout {
         property real keyWeight: 154
 
         SymbolKey {
-            weight: 217
+            weight: availableLanguageLayouts.length === 1? 217 : 108.5
```
Comment (AndreaRicchi) [inline src/qml/CsLayout.qml:230]: This code is not formatted! Also all the other layouts.

File: src/qml/InputPanel.qml [hunk 2]
```diff
@@ -135,6 +142,13 @@ Item {
             }
         }
 
+        Connections{
+            target: InputPanel
+            onLanguageLayoutChanged:{
```
Comment (AndreaRicchi) [inline src/qml/InputPanel.qml:147]: Same: not formatted

File: src/qml/LanguageKey.qml
Comment (AndreaRicchi) [inline src/qml/LanguageKey.qml:1]: Same: not formatted

**Review by AndreaRicchi — CHANGES_REQUESTED** [review]
Thanks for your contribution, **this is a great improvement**. Some minor comments regarding code formatting. Also, I suggest squashing the two commits for a cleaner history.

**Comment by Johannes-bichler** [comment]
@AndreaRicchi How do you want this code to be formatted? Do you recommend any formatter for this purpose?

File: src/qml/CsLayout.qml (hunk 1 above)
Comment (panicking) [reply src/qml/CsLayout.qml:230]: @Johannes-bichler I think this one from
            weight: availableLanguageLayouts.length === 1? 217 : 108.5
to
            weight: availableLanguageLayouts.length === 1 ? 217 : 108.5

File: src/qml/InputPanel.qml (hunk 2 above)
Comment (panicking) [reply src/qml/InputPanel.qml:147]: @Johannes-bichler from  Connections{ to  Connections {

**Comment by Johannes-bichler** [comment]
Formatting has been completed and commits were squashed, everything should look fine now.

File: src/qml/CsLayout.qml (hunk 1 above)
Comment (Johannes-bichler) [reply src/qml/CsLayout.qml]: @panicking I have used the QtCreator included QML Formatter and it worked exactly like you proposed. :grin:

**Review by AndreaRicchi — APPROVED** [review]

**Comment by AndreaRicchi** [comment]
@Johannes-bichler you can also use the QML formatter included in the Qt SDK!

**Pull request merged**